chapters/offenses.adoc

```diff
@@ -27,126 +98,155 @@ The ball must have moved at least 0.05 meters to be considered as <<Ball In And
+speed limit is to avoid robots harming the people on the field.
+
+===== Multiple Defenders
+NOTE: This rule does not use the standard sanctions defined for <<Fouls, fouls>>.
+
+If a robot other than the keeper touches the ball while being partially inside its own defense area, the game is stopped, the robot receives a <<Yellow Card, yellow card>> and the opponent team resumes the game with a <<Free Kick, free kick>>. The foul counter is not increased.
+
+If a robot other than the keeper touches the ball while being entirely
+inside its own defense area, the game is stopped and a <<Penalty Kick,
+penalty kick>> is awarded to the other team. The foul counter is not
+increased.
+
+===== Boundary Crossing
+A robot must not kick the ball over the field boundary such that the ball leaves the field.
+
+
+==== No Stop Fouls
+
+Violations in this section are fouls, but they do not cause a <<Stop,
+stop>>. Instead, the game continues but time is added to a per-team no
+goal timer. Any goal scored by the team while this timer has time
+remaining is <<Scoring Goals, considered invalid>>.
+
+Each violation adds 2 seconds to the timer. The timer is cleared if
+the opposing team scores a <<Scoring Goals, valid goal>> causing a
+<<Kick-Off, kick-off>>. The timer is also cleared at half-time and at
+overtime.
+
+The same no stop foul cannot be triggered again until the foul
+condition has stopped being violated or there has been 1 second since
+the foul was first triggered. This is to allow teams to adjust their
+robots' positions, ball speed or any other property that is causing
+the violation before being penalized additional times.
+
+===== Attacker In Defense Area
+A robot must not touch the ball while being partially or fully inside the opponent <<Defense Area, defense area>>.
+
+NOTE: When the ball is <<Ball In And Out Of Play, out of play>>, the more strict rule <<Robot Too Close To Opponent Defense Area>> applies instead.
+
+===== Excessive Dribbling
+A robot must not <<Dribbling Device, dribble>> the ball further than 1 meter, measured linearly from the ball location where the dribbling started. A robot begins dribbling when it makes contact with the ball and stops dribbling when there is an observable separation between the ball and the robot.
+
+NOTE: Dribblers can still be used to dribble large distances with the ball as long as the robot periodically loses possession, such as kicking the ball ahead of it as human soccer players often do.
 
-==== Defender Too Close To Ball
-A robot's distance to the ball must be at least 0.5 meters during an opponent <<Kick-Off, kick-off>> or <<Free Kick, free kick>>.
+===== Ball Speed
+A robot must not accelerate the ball faster than 6.5 meters per second in 3D space.
+
+===== Defender Too Close To Ball
+A robot's distance to the ball must be at least 0.5 meters during an opponent <<Kick-Off, kick-off>>, <<Free Kick, free kick>>.
 The game is resumed with the same command that was issued before the foul.
```

Comment:
Obsolete, if the game state does not change?

Also, this has more implications.
It's easy, especially with mediocre vision, to misidentify the ball to be "in play".
The previous rules already punished teams for misidentifying with a foul - so that's not the point.

The problem is that afaik in play / out of play has to implemented by the teams and therefore the "Stop" command to push each AI towards a stable "The ball is out of play, keep your distance" is necessary.

Is there any other hint to the AI they have to keep their distance right now?

Reply:
I've remove the line.

Teams have to do the following:
* If command is stop or prepare_kickoff -> keep distance
* If command is normal_start or direct_free_kick -> wait until ball moved or timeout counter got below 0 (contained in referee msg)

Now that we do not stop for this rule, what can happen?
* Opponent is just a bit too close to the ball -> team is punished, attacking team should have no significant disadvantage
* Opponent is approaching the ball -> team is punished, attacking team can not perform free kick -> should be repeated

The second case often happens due to vision issues, which the autoRefs could also suffer from. So the human ref should have an eye on these situations anyway.
I've added another sentence that allows the human ref to repeat the situation. I'd like to avoid defining this more concrete, to keep it simple in the rules and in the autoRefs.